tests/unit/window/test_window_new.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import arcade
+
+
+class TestView(arcade.View):
```

Comment:
This unit tests needs to use the global test window instead of making a new one or this will be extremely complicated.

Write the tests like this. You will automatically get a test window fixture you can use. Creating new windows are not allowed in our unit tests. You should get a "clean" window for every function.
```py
def test_something(window):
    window.run(..)
```